packages/gatsby/src/commands/develop.ts

```diff
@@ -155,6 +155,14 @@ class ControllableScript {
     }
     this.process.on(`exit`, callback)
   }
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  send(msg: any): void {
+    if (!this.process) {
+      throw new Error(`Trying to attach exit handler before process starter`)
```

Comment:
NIT :)
```suggestion
      throw new Error(`Trying to attach exit handler before process started`)
```

Reply:
And well, this msg doesn't make sense - we don't try to attach exit handlers - is this copied from somewhere? :P